vm-persistence.md

```diff
@@ -0,0 +1,107 @@
+## VM persistence
+
+### Preparation
+
+To persist a VM, the `Persistent Virtual Machine` add-on must be installed
+first.
+
+```bash
+git clone https://github.com/petrkotas/virt-vmconfig-crd
```

Comment:
I'd not like to reference personal repositories in the user-guide, instead I'd like to see that we get it to the incubator where we have an at least lean review of what is going in.

Thus once it's there we can happily add this to the official documentation.